pkg/common/types.go

```diff
@@ -15,10 +15,19 @@ type SMIPolicy struct {
 	// TODO: include actual SMI policy
 }
 
-// Runnable is a type of generic function that can be executed and it will return pass/fail on a given check.
+// Runnable is a type of generic function that can be executed; it returns pass/fail on a given check.
 type Runnable interface {
+	// Run executes a check.
 	Run() error
+
+	// Info returns human-readable information on what check is being executed.
 	Info() string
+
```

Comment:
Can we also have another one called 'MoreInfo() string` or something like that? The event and log checks that I worked on have messages such as "to have **_more info_** on this error, please run `kubectl ...`"

Reply:
I'd love to understand what the intention is. The Suggestion() may cover this scenario.